### ISSUE TITLE
Path to included my_custon.cnf has changed

### ISSUE THREAD

**Issue by ryanbrownellbounteous** [lando/mysql#24]
I have been trying to modify the MySQL settings for a container using MySQL 5.7. I have not been able to get anything to work.

I investigated the settings file located at `/opt/bitnami/mysql/conf/my.cnf` and found that the location specified in `services/mysql/builder.js` no longer matches the path for including custom `.cnf` files.

The new path appears to be `/opt/bitnami/mysql/conf/bitnami/my_custom.cnf`

**Comment by ryanbrownellbounteous** [comment]
I found a work-around for this issue by doing the following

```
services:
    database:
    type: mysql
    remoteFiles:
      database: '/opt/bitnami/mysql/conf/bitnami/my_custom.cnf'
```

In general there appears to be significant differences between implementation and documentation for this recipe.

**Comment by reynoldsalec** [comment]
In my testing the file `/opt/bitnami/mysql/conf/my_custom.cnf` gets copied to `/opt/bitnami/mysql/conf/bitnami/my_custom.cnf` and is read as config my MySQL. So the existing setup in the Lando builder works, but I'm actually not sure where/why that copying occurs. Tested both on a stand-alone MySQL service and with the Acquia recipe.

Looking at Bitnami's [docs on configuring the MySQL image](https://hub.docker.com/r/bitnami/mysql#configuration), it seems like they still want you to put your cnf file at `/opt/bitnami/mysql/conf/my_custom.cnf`, so I don't think the Lando config is out-of-date. Based on that info, I'm guessing any copying to an alternative location is performed by the Bitnami Dockerfile. Going to be happy when we're completely off of Bitnami for this reason and more :)

@ryanbrownellbounteous super curious why your config wasn't loading, it you ever want to chat about it DM me on the Lando Slack: https://launchpass.com/devwithlando

tl;dr for other folks, existing configuration and doc instructions should work. If you're not seeing your .cnf file load, you either need to `lando rebuild` or double-check that you're [correctly adding the cnf file to your `.lando.yml` settings](https://docs.lando.dev/mysql/config.html#using-a-custom-mysql-config-file).
